sparkbench/common/pom.xml

```diff
@@ -28,4 +28,18 @@
       <version>${hadoop.mr2.version}</version>
     </dependency>
   </dependencies>
+  <profiles>
```

Comment:
This profile can be promoted to upper level: sparkbench/pom.xml for sharing between different submodules?

Reply:
Done